tests/test_mask.py

```diff
@@ -13,13 +13,23 @@ def fixture_testvol():
     return read_dicoms(os.path.join(os.path.dirname(__file__), "testdata"))[0]
 
 
-def test_apply(fixture_testvol):
-    res = apply(fixture_testvol)
+def test_LMInferer(fixture_testvol):
+    inferer = LMInferer(
```

Comment:
You should probably need to add a test for the case where a `modelpath` is provided

Reply:
That's right, added a test to cover this. Thanks a lot for your help!